### DISCUSSION TITLE
Resizable Editor Windows

### DISCUSSION THREAD

**Discussion by edmundito** [godotengine/godot-proposals#9266] [Editor]
Hello, one of the issues I'm running into is that I can run Godot on my laptop or connect to a screen, but the laptop display has a higher resolution than my display. So what happens is that the height in windows within Godot are too long, and I cannot resize them.

![image](https://github.com/godotengine/godot-proposals/assets/168664/7eef25de-d74a-4489-b11e-9ad52eafec83)

In this scenario, I cannot:
1. Move the window up
2. Resize the window down (it only resizes the main window)

A possible workaround would be for me to:
1. Disconnect the laptop from the display
2. Open the laptop
3. Resize the window.

It would be nice to find some way to resize the window somehow. One way this is done on Mac is to double-click on the title bar to readjust the window. Alternatively, Godot could detect that the window is too large and try to resize it to fit the main window.

**Comment by Calinou** [comment]
Do you have **Single Window Mode** enabled in the Editor Settings? By default, windows are fully independent from each other so they can be resized freely. macOS defaults to client-side decorations though (at least for the main window), so Godot draws the window borders instead of the OS doing it. You can disable client-side decorations in the Editor Settings by unchecking **Interface > Editor > Expand to Title**.
